cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/AwsPlatformResources.java

```diff
@@ -841,50 +842,48 @@ public CloudEncryptionKeys encryptionKeys(ExtendedCloudCredential cloudCredentia
         CloudEncryptionKeys cloudEncryptionKeys = new CloudEncryptionKeys(new HashSet<>());
         AwsCredentialView awsCredentialView = new AwsCredentialView(cloudCredential);
         AmazonKmsClient client = awsClient.createAWSKMS(awsCredentialView, region.value());
+        Set<KeyListEntry> kmsKeys = getListKeyResult(client);
```

Comment:
(nit) Can `kmsKeys` be made as `listKeysResult` for naming consistency? If you do that, you may see less differences in the PR, as a bonus.